electron-builder.yml

```
@@ -36,6 +36,12 @@ mac:
   hardenedRuntime: true
   gatekeeperAssess: false
   notarize: false
+  target:
+    target: 'default'
+    arch: [
+      'x64',
+      'arm64'
+    ]
```

Use YAML array syntax

```suggestion
    arch: 
      - x64
      - arm64
```